src/nanoemoji/nanoemoji.py

```diff
@@ -190,6 +195,77 @@ def _not_impl(*_):
     raise NotImplementedError("%s not implemented" % FLAGS.color_format)
 
 
+def _draw(source: SVGPath, dest: Glyph, svg_units_to_font_units: Affine2D):
+    pen = TransformPen(dest.getPen(), svg_units_to_font_units)
+    skia_path(source.as_cmd_seq()).draw(pen)
+
+
+def _next_name(ufo: ufoLib2.Font, name_fn) -> str:
+    i = 0
+    while name_fn(i) in ufo:
+        i += 1
+    return name_fn(i)
+
+
+def _glyph(color_glyph: ColorGlyph, painted_layer: PaintedLayer) -> Glyph:
```

Comment:
I find the name of this function not very informative. I got the idea, when you name a method with just a noun, that you want to direct the reader's attention to its return value, i.e. what is produced by that method, in this case a UFO Glyph. But in this case the function also has side effects that are not conveyed by the name, like adding the new glyph to the UFO and drawing the painted layer onto that glyph. In fact, whe you use this in `_glyf_ufo` function, you are discarding the return value and only calling the function for its side-effects.
Could you perhaps name it so that it has some verb in it to make it clearer that the function is _doing_ something with the parameters, besides converting input to output?
Maybe the fact that there are so many "glyph" in the signature of this function that make it not immediately obvious what is what.

Reply:
I have somewhat uncreatively named in `_create_glyph`. +1 to too many Glyph's; perhaps ColorGlyph needs a new name, though I don't immediately have one to offer.